hack/fork_exec_snoop.bt

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bpftrace
+
+//   Copyright 2022 Nicolas Hillegeer <nicolas@hillegeer.com>
+//   Modifications Copyright 2023 Paul Wallrabe <pwallrab@redhat.com>
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//   This file is based on the work of Nicolas Hillegeer <nicolas@hillegeer.com>
+//   and has been modified by Paul Wallrabe <pwallrab@redhat.com> to add support
+//   for normalized timestamps and implemented a trigger to start printing to
+//   console once a certain condition is reached specific to Podman.
+//
+// Traces processes that are created using the fork/exec model.
+// Output is printed after the first command containing "podman" is executed.
+//
+// Usage: sudo ./fork_exec_snoop.bt
```

Comment:
I need to run it via `sudo bpftrace ./hack/fork_exec_snoop.bt`

Reply:
good catch! `chmod +x` does the trick. I will update the permissions on the file.